src/logion/controllers/adapters/locrequestadapter.ts

```diff
@@ -139,6 +149,26 @@ export class LocRequestAdapter {
             ...description
         };
     }
+
+    async getSelectedIssuersIdentities(selectedIssuers: VerifiedThirdPartySelectionAggregateRoot[]): Promise<VerifiedIssuerIdentity[]> {
+        const identities: VerifiedIssuerIdentity[] = [];
+        for(const selectedIssuer of selectedIssuers) {
+            const issuer = selectedIssuer?.issuer || "";
+            if(selectedIssuer?.identityLocId) {
```

Comment:
It's not clear to me under which condition `selectedIssuer?.identityLocId` could be undefined. According to relational model, the column `vtp_loc_id` remains not nullable.

Reply:
I will never be, but the compiler does not know that. By explicitly checking that the value is defined, the compiler leaves me alone. ;-)